chess/Model/Piece.swift

```diff
@@ -7,22 +7,24 @@
 
 import Foundation
 
-enum PieceType: String {
-    case pawn = "P"
-    case rook = "R"
-    case knight = "N"
-    case bishop = "B"
-    case queen = "Q"
-    case king = "K"
-}
+struct ChessPiece {
 
-enum PieceColor: String {
-    case white = "W"
-    case black = "B"
-}
+    enum PieceKind: String {
```

Comment:
just `Kind`. Check an example on swift docs https://docs.swift.org/swift-book/documentation/the-swift-programming-language/nestedtypes/

It is a common pattern when you omit key word (`Piece` in your case) since you are using nested type. So you can access it only by using `ChessPiece.Kind` notation